oci/oci.go

```diff
@@ -17,6 +17,21 @@ import (
 	spec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
+// BlobInfo represents the mutable properties of a blob
```

Comment:
Can we consider moving these structures to the `content` pkg?
Initially, `oci` pkg was intended to be a generic package without containerd-specific stuff.